Booked patient appointments eir

dominickasanga opened the pull request:
get git push booked_patient_appointment_eir

dominickasanga committed:
Merge branch 'chore/find_patients_pagination' of https://github.com/HISMalawi/BHT-EMR-API into chore/find_patients_pagination

dominickasanga committed:
modified:   app/controllers/api/v1/program_appointments_controller.rb
odified:   app/services/program_appointment_service.rb
modified:   config/routes.rb
new file:   vendor/cache/activejob-uniqueness-0.3.1.gem
new file:   vendor/cache/redlock-2.0.6.gem

dominickasanga committed:
modified:   app/controllers/api/v1/program_appointments_controller.rb
modified:   app/services/program_appointment_service.rb
modified:   config/routes.rb

fchiyenda committed:
Merge remote-tracking branch 'origin/chore/find_patients_pagination' into booked_patient_appointments_eir

fchiyenda reviewed (changes requested):
I think we can use next appointment date we can have an engine

dominickasanga committed:
modified:   app/controllers/api/v1/program_appointments_controller.rb
modified:   app/services/appointment_service.rb
new file:   app/services/immunization_service/appointment_engine.rb
modified:   app/services/program_appointment_service.rb
modified:   config/routes.rb

dominickasanga committed:
Merge branch 'booked_patient_appointments_eir' of https://github.com/HISMalawi/BHT-EMR-API into booked_patient_appointments_eir

dominickasanga committed:
modified:   app/services/immunization_service/appointment_engine.rb
